airbyte-integrations/connectors/source-snowflake/acceptance-test-config.yml

```
@@ -1,7 +1,29 @@
 # See [Source Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/source-acceptance-tests-reference)
 # for more information about how to configure these tests
 connector_image: airbyte/source-snowflake:dev
-tests:
+acceptance_tests:
```

```suggestion
test_strictness_level: high
acceptance_tests:
```
I'd be curious to check the test output under high test strictness level, we might get this for free.